Extend/Language.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ *
+ */
+
+namespace Wirecard\Oxid\Extend;
+
+/**
+ * Extends the OXID Language class
+ *
+ * @mixin \OxidEsales\Eshop\Core\Language
+ *
+ * @since 1.0.1
+ */
+class Language extends Language_parent
+{
+    const TRANSLATION_KEY_PREFIX = 'wdpg_';
+    const FALLBACK_LANGUAGE_ABBR = 'en';
+
+    /**
+     * @var int
+     *
+     * @since 1.0.1
+     */
+    private $_iFallbackId = null;
+
+    /**
+     * @inheritdoc
+     *
+     * Provides a fallback language string if the chosen language is not available for the plugin
+     *
+     * @param string $sStringToTranslate Initial string
```

Comment:
1) The "sStringToTranslate" param description can be omitted as the variable name is already fully descriptive.

2) Also, would it be more intuitive to rename "iLand" to `iLanguageId`?

3) Please, rename "blAdminMode" to `bAdminMode` in order to stay consistent within our plugin.

4) Additionally, we should have consistent param descriptions starting with an upper-case letter.